website/client/src/assets/scss/button.scss

```diff
@@ -3,23 +3,25 @@
   font-family: 'Roboto', sans-serif;
   font-size: 14px;
   font-weight: bold;
-  line-height: 1.71;
-  border: 1px solid transparent;
-  padding: 4px 12px;
+  line-height: 1.714;
+  border: 2px solid transparent;
+  padding: 2px 12px;
   border-radius: 4px;
   box-shadow: 0 1px 3px 0 rgba($black, 0.12), 0 1px 2px 0 rgba($black, 0.24);
   color: $white;
 
   &:hover, &:focus {
-    box-shadow: 0 3px 6px 0 rgba($black, 0.16), 0 3px 6px 0 rgba($black, 0.24);
+    box-shadow: 0px 3px 6px 0px rgba(26, 24, 29, 0.16), 0px 3px 6px 0px rgba(26, 24, 29, 0.24);
```

Comment:
No need to use units when the value is 0. We should also aim to be consistent across the code base. If you look above at line 10, you'll see that 0 does not have a unit and the `box-shadow` property is using the `$black` variable which is the same as `rgb(26, 24, 29)`.